src/util/alloc/bumpallocator.rs

```diff
@@ -41,47 +37,6 @@ impl<VM: VMBinding> BumpAllocator<VM> {
         self.reset();
         self.space = space;
     }
-
-    pub fn scan(&self) {
```

Comment:
Two reasons:
1 - it's not used anywhere.
2 - it uses scan_region which is removed because of using old object model methods.

If it is useful, I can update its implementation and put it back